esp/useful_scripts/csv_student_payment.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python2
+#
+# Add accounting records for students that have already paid (e.g. on an external website).
+# Assumes that a student has paid for the entirety of their amount due.
+#
+# Requires the name of the program and a csv that has a column named 'username'
+# (which contains student usernames or user ids). Any other columns will be ignored.
+#
+
+from script_setup import *
+
+from esp.accounting.models import *
+from esp.users.models import ESPUser
+from esp.accounting.controllers import IndividualAccountingController
+
+import csv
+import os
+import argparse
+
+from datetime import datetime
+
+
+parser = argparse.ArgumentParser(description = "Add accounting records for students that have already paid")
```

Comment:
Usually good practice is to wrap the entire script in
```
def main():
    ...
```
and then at the end you do
```
if __name__ == '__main__':
    main()
```
so that you can import the script without running it.